app/core/Router.coffee

```diff
@@ -13,7 +13,24 @@ module.exports = class CocoRouter extends Backbone.Router
     @initializeSocialMediaServices = _.once @initializeSocialMediaServices
 
   routes:
-    '': go('HomeView')
+    '': ->
+      # Testing new home page
+      group = me.get('testGroupNumber') % 4
+      group = 0
+      switch group
+        when 0 
+          group = 'control'
+          @routeDirectly('HomeView', [])
+        when 1
+          group = 'home-with-note'
+          @routeDirectly('HomeView', [], { withTeacherNote: true })
+        when 2
+          group = 'new-home-student'
+          @routeDirectly('NewHomeView', [], { jumbotron: 'student' })
+        when 3
+          group = 'new-home-characters'
+          @routeDirectly('NewHomeView', [], { jumbotron: 'characters' })
+      application.tracker.identify newHomePageGroup: group unless me.isAdmin()
```

Comment:
Homepage treated as one word instead of two everywhere else.  We should be consistent.  `newHomepageGroup`